desktop/api/src/main/java/org/datacleaner/util/UserWindowSizePreferencesUtils.java

```diff
@@ -0,0 +1,88 @@
+/**
+ * DataCleaner (community edition)
+ * Copyright (C) 2014 Neopost - Customer Information Management
+ *
+ * This copyrighted material is made available to anyone wishing to use, modify,
+ * copy, or redistribute it subject to the terms and conditions of the GNU
+ * Lesser General Public License, as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License
+ * for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this distribution; if not, write to:
+ * Free Software Foundation, Inc.
+ * 51 Franklin Street, Fifth Floor
+ * Boston, MA  02110-1301  USA
+ */
+package org.datacleaner.util;
+
+import java.awt.Dimension;
+import java.util.Map;
+
+import org.datacleaner.user.UserPreferences;
+
+public class UserWindowSizePreferencesUtils {
+
+    private final UserPreferences _userPreferences;
+    private final String _identifier;
+    private final int _defaultWidth;
+    private final int _defaultHeight;
+
+    public UserWindowSizePreferencesUtils(final UserPreferences userPreferences, final String identifier,
```

Comment:
Can we add a javadoc. I don't know what "identifier" is - I imagine it's a name/ID of the window?